data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,435 @@
+				`	"Unfortunately, I did find something."`
+			`	You hand Sayari a data stick, "I have saved everything from my logs on there."`
+			`	Sayari looks crestfallen and takes the data stick gingerly, looking at it as if it were some snake that might bite her at any moment.`
+			`	"I will have a look," is all she says.`
+			`	No sooner has she pocketed it than there is a knock at the door and an aide looks in, "The Council will see you now."`
+			`	The next twenty minutes prove to be an exercise in tedium as various members of the council interrogate you over apparently mundane details of your recent actions as recorded in the FOI report. You're not really sure what most of it has to do with anything, but eventually one of the elders stands up.`
+			`	"I think we've generally had enough of this here, answer me two questions and we shall make our decision," he says.`
+			`	"Do you think that a diplomatic post should be able to defend itself? Or should it trust to the strength of those who meet there to ensure its protection?"`
+			`	This seems like a very strange question...`
+			choice
+				`	"It should be strong enough to defend itself. You cannot trust your enemy."`
+					goto trust
+				`	"Both sides must be trusted to protect a diplomatic post, or there is no point to having one."`
+					goto caution
+			label trust
+			apply
+				set haitrust
+			`	You seem to get no reaction and the Elder follows up with, "What is the gravest threat, in your opinion, facing humanity?"`
+				goto return
+			label caution
+			apply
+				set haicaution
+			`	You seem to get no reaction and the Elder follows up with, "What is the gravest threat, in your opinion, facing humanity?"`
+			label return
+			choice
+				`	"The Pug"`
+				`	"The Alphas"`
+				`	"Unknown Jump-Capable Aliens"`
+				`	"Experimental Technologies"`
+				`	"Hubris"`
+					goto wise
+			`	There is a collection of chittering laughs from the various council members. You're not sure if there was something wrong with what you said but you aren't left long to ponder.`
+			`	"It seems the human agent is an honest fool, but an honest fool can at least be trusted to be foolish honestly on all fronts."`
+			`	You're pretty sure you should be offended by that, but the councilmembers are smiling and don't look like they're about to evict you.`
+				goto end
+			label wise
+			`	"It seems you learnt the correct lesson from the crisis of your species," he says.`
+			`	You see several appreciative nods happening around the room out of the corner of your eyes.`
+			label end
+			`	"You may act as the oversight for the human governments in this project. Proposal D-86513-7P4 is approved. Council adjourned."`
+			`	When you meet Sayari afterwards a touch of her usual enthusiasm has returned.`
+			`	"This is fantastic news, now all that remains to be done is for you to escort the construction fleet from Makerplace to Ultima Thule and observe the initial stages. They'll be waiting for your arrival on Makerplace."`
```

Comment:
Just a small continuity thing, Sayari wouldn't know how the council meeting went as she was in another room, maybe this could be fixed by saying 'When you meet back with Sayari in the antechamber and tell her the good news, her face lights up with enthusiasm'